installers/community/arm64/edge/start.cmd

```diff
@@ -0,0 +1,31 @@
+@echo off
+set COMPOSE_FILE=docker-compose.yml
+echo "Using compose file: %COMPOSE_FILE%"
+
+echo  " Starting.. %COMPOSE_FILE%"
+docker-compose -f "%COMPOSE_FILE%" up
+
+echo "\n"
+echo  "Installing cors plugin"
+curl -X POST http://localhost:8001/plugins/ ^
+--data "name=cors"  ^
+--data "config.origins=*" ^
```

Comment:
Where will this run? Is `*` appropriate for all of those locations?

Reply:
Commented out this code as it is not in use right now. But noted the *, it should be parameterized at the very least.